File: count_div/count_div.py
# https://app.codility.com/demo/results/trainingHQGYQS-7YP/


def solution(A, B, K):
    divs_until_lower_bound = A // K
    divs_until_upper_bound = B // K
    increment = 1 if A % K == 0 else 0

    return divs_until_upper_bound - divs_until_lower_bound + increment

File: count_div/count_div.js
// https://app.codility.com/demo/results/trainingHQGYQS-7YP/

function solution(A, B, K) {
  const divsUntilLowerBound = Math.floor(A / K);
  const divsUntilUpperBound = Math.floor(B / K);
  const increment = A % K == 0 ? 1 : 0;

  return divsUntilUpperBound - divsUntilLowerBound + increment;
}
